R/score_model_out.R

```diff
@@ -125,7 +125,7 @@ score_model_out <- function(model_out_tbl, oracle_output, metrics = NULL,
 #' Get scoring metrics
 #'
 #' @param forecast A scoringutils `forecast` object (see
-#' [scoringutils' general information on creating a `forecast` object][scoringutils::as_forecast_doc_template()]
+#' [scoringutils' general information on creating a forecast object][scoringutils::as_forecast_doc_template()]
```

Comment:
This is unrelated to the main point of this PR, but fixes a complaint that `check` raised: `:heavy_multiplication_x: score_model_out.R:127: @param markdown links must contain plain text.`